packages/block-library/src/breadcrumbs/index.php

```diff
@@ -100,9 +100,25 @@ function render_block_core_breadcrumbs( $attributes, $content, $block ) {
 			$show_terms = $attributes['prefersTaxonomy'];
 		}
 
-		// Build breadcrumb trail.
+		// Add post type archive link if applicable.
+		$post_type_object = get_post_type_object( $post_type );
+		$archive_link     = get_post_type_archive_link( $post_type );
+		if ( $archive_link && untrailingslashit( home_url() ) !== untrailingslashit( $archive_link ) ) {
```

Comment:
What's the chance the archive link doesn't have trailing slashes? I'm considering why the removal of the trailing slashes is needed here, and if it would be shorter to do just:


```suggestion
		if ( $archive_link && home_url( '/' ) !== $archive_link ) {
```

Reply:
>What's the chance the archive link doesn't have trailing slashes?

Probably high, because my default setup actually didn't have the trailing slash.

Also see https://github.com/WordPress/gutenberg/pull/73435#discussion_r2543356620. In general there are many filters applied and even this might not be failproof . We should keep an eye in the future if some edge cases or different setups with filters etc.. occur.